### PR TITLE
added pramaters for time span and confidence threshold

### PR DESCRIPTION
**Pull request by WaxXwing**
For #821

### CODE REVIEW

File: scripts/generate_sheet.py [hunk 1]
```diff
@@ -53,12 +53,18 @@ def generate_sheet(model_name, token):
...
 
 def main():
-    description = "Perform evaluation on bugs from last week on the specified model and generate a csv file "
+    description = "Perform evaluation on bugs from specified days back on the specified model and generate a csv file "
     parser = argparse.ArgumentParser(description=description)
 
     parser.add_argument("model", help="Which model to generate a csv for.")
     parser.add_argument("token", help="Bugzilla token")
+    parser.add_argument(
+        "span", type=int, help="No. of days back from which bugs will be evaluated"
```
Comment (marco-c) [inline scripts/generate_sheet.py]: Let's use `days` as a name instead of `span`

File: scripts/generate_sheet.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 from bugbug.models import get_model_class
 
 
-def generate_sheet(model_name, token):
+def generate_sheet(model_name, token, span, threshold):
```
Comment (marco-c) [inline scripts/generate_sheet.py]: And let's call the variable days instead of span.

Comment (WaxXwing) [reply scripts/generate_sheet.py]: ok, making all the `span` s into `days`

File: scripts/generate_sheet.py [hunk 3]
```diff
@@ -53,12 +53,18 @@ def generate_sheet(model_name, token):
...
     parser = argparse.ArgumentParser(description=description)
 
     parser.add_argument("model", help="Which model to generate a csv for.")
     parser.add_argument("token", help="Bugzilla token")
+    parser.add_argument(
+        "days", type=int, help="No. of days back from which bugs will be evaluated"
+    )
+    parser.add_argument(
+        "threshold", type=float, help="Confidence threshold for the model"
+    )
```
Comment (marco-c) [inline scripts/generate_sheet.py]: Forgot to tell you before: let's use 7 as default for the days, and 0.7 as default for the threshold (to keep the same behavior as before if the arguments are not passed).

**Pull request merged**